README.md

```diff
@@ -119,6 +119,55 @@ grunt.initConfig({
 
 One might target the task using `grunt parallel:assets`. This would run compass, requirejs, and a custom shell script at the same time, each logging to your console when they are done.
 
+#### Expose grunt.config properties
+
+Sometimes you want some properties from grunt.config to persist in the spawned task. To do this you can put `exposeGruntConfigKeys: <Array of config props>` in your tasks configuration, and grunt-parallel will apply the given properties to the spawned grunt process.
```

Comment:
Thanks for providing a more realistic example. I think now I finally understand the use case :) .
Seem that the main part here is that config values can be changed dynamically (by other tasks) - and special handling is needed for that. Constant config values are always available to child processes as is, without any extra configuration.
I feel it's important enough to be explicitly mentioned in the README.
What do you think?